R/Sql.R

```diff
@@ -494,7 +494,7 @@ executeSql <- function(connection,
       tryCatch(
         {
           startQuery <- Sys.time()
-          lowLevelExecuteSql(connection, sqlStatement)
+          DBI::dbExecute(connection, sqlStatement)
```

Comment:
In v6.0.0, dbExecute will apply translation, whereas lowLevelExecuteSql will not. So I think this will break `executeSql` when using a DatabaseConnectorConnection? The `executeSql()` function probably shouldn't apply any translation (for backwards compatibility).

Reply:
Can we allow users to opt out of translation when calling dbExecute or dbGetQuery?

Reply:
Do I need to fix failing tests?

Reply:
For all DBI functions that in 6.0.0 automatically perform translation we could add a `translate` logical argument, that defaults to TRUE. Then, in querySql and executeSql we could call with `translate = FALSE`. But I'm not sure what happens when for example calling `executeSql()` for a non-DatabaseConnector-DBI with `translate = FALSE`. Would it throw a warning about the unused argument?